src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const price = days * PRICE_PER_DAY;
+
+  if (days >= 7) {
+    return price - 50;
+  }
+
+  if (days >= 3) {
+    return price - 20;
+  }
+
+  if (days < 3) {
+    return price;
+  }
```

Comment:
rewrite your code following the checklist
<img width="726" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/70220667/b0643cf7-7978-4d66-8284-683a990567b1">